argilla/docs/how_to_guides/distribution.md

```diff
@@ -77,4 +77,61 @@ dataset = client.datasets("my_dataset")
 dataset.settings.distribution.min_submitted = 4
 
 dataset.update()
-```
\ No newline at end of file
+```
+
+## Track your team's progress
+
+You can check the progress of the annotation task by using the `dataset.progress` method.
+This method will return the number of records that have the status `completed`, `pending`, and the
+total number of records in the dataset.
+
+```python
+import argilla as rg
+
+client = rg.Argilla(api_url="<api_url>", api_key="<api_key>")
+
+dataset = client.datasets("my_dataset")
+
+progress = dataset.progress()
+```
+```json
+{
+    "total": 100,
+    "completed": 10,
+    "pending": 90
+}
+```
+
+You can see also include to the progress the users distribution by setting the `with_users_distribution` parameter to `True`.
+This will return the number of records that have the status `completed`, `pending`, and the total number of records in the dataset,
```

Comment:
```suggestion
This will return the number of records that have the status `completed` and `pending` for each user, and the total progress of records in the dataset. A user's `completed` records have been reviewed by enough team users to satisfy the distribution settings. A user's `pending` records have been reviewed by the user but require further users to satisfy the distribution settings.
```

Reply:
@nataliaElv Do you think this is useful? I'm just not sure the user will understand the difference between pending and complete.

Reply:
Ok. I guess my addition complicates it further. The link should be enough.